### DISCUSSION TITLE
API deprecation

### DISCUSSION THREAD

**Discussion by tmilnthorp** [angularsen/UnitsNet#1027] [General]
Should we deprecate the following static properties from the quantities:
- BaseDimensions
- BaseUnit
- Units

These are all accessible from the Info property/classes now.

**Comment by tmilnthorp** [comment]
@angularsen

**Reply by angularsen** [reply]
- Units: I'm not so sure. The replacement would be `Length.Info.UnitInfos.Select(ui => ui.Value)`, which is a lot more cumbersome than just `Length.Units`. The arrays are small so I don't worry about the memory usage either, but we could add a getter that enumerated the UnitInfos in the same way. Another alternative is to un-obsolete `Length.Info.Units` property, that could be a fair compromise and keep most things in one place. 
- `Length.Zero` should still remain I think, like `TimeSpan.Zero`.
- Min/MaxValue are already moved and obsoleted (removed in v5).
- BaseDimensions/BaseUnit: Sure.

**Comment by angularsen** [comment]
I'll get to this soon
